src/locales/cy/translation.json

```diff
@@ -863,9 +863,15 @@
       },
       "sqae3L7gOdizeRqFMw_KCDlhcyg": {
         "header": "Repay and manage benefit money you owe",
-        "description": "Ad-dalu a rheoli arian budd-dal sy’n ddyledus gennych.",
+        "description": "Ad-dalu a rheoli arian budd-dal sy'n ddyledus gennych.",
+        "link_text": "Go to your repay and manage benefit money you owe account",
```

Comment:
Just to confirm, do we not have the Welsh language translation yet? 
Or is it not relevant at this stage and can be added later?

Reply:
Thanks Dana, yes we don't have the full welsh translations. I would think its not relevant as they are currently testing